libraries/helpers.rb

```diff
@@ -148,11 +145,12 @@ def init_records_script
         mkdir /tmp/#{mysql_name}
 
         cat > /tmp/#{mysql_name}/my.sql <<-EOSQL
-DELETE FROM mysql.user ;
-CREATE USER 'root'@'%' IDENTIFIED BY '#{Shellwords.escape(new_resource.initial_root_password)}' ;
-GRANT ALL ON *.* TO 'root'@'%' WITH GRANT OPTION ;
+UPDATE mysql.user SET Password=PASSWORD('#{Shellwords.escape(new_resource.initial_root_password)}');
+DELETE FROM mysql.user WHERE user LIKE '';
+DELETE FROM mysql.user WHERE user = 'root' and host NOT IN ('127.0.0.1','localhost');
+DROP DATABASE test;
+DELETE FROM mysql.db WHERE db LIKE 'test%'
```

Comment:
I'm still trying to figure out how to use git as well. I wasn't sure how to exclude showing that change